.pipelines/cni/k8s-e2e/k8s-e2e-step-template.yaml

```diff
@@ -0,0 +1,57 @@
+parameters:
+  testName: ""
+  name: ""
+  clusterName: ""
+  ginkgoFocus: ""
+  ginkgoSkip: ""
+  os: ""
+  processes: "" # Number of parallel processes
+  attempts: ""
+
+
+steps:
+  - script: |
+      set -ex
+
+      # ginkgoSkip cant handle only |LinuxOnly. Need to have check
+      if ${{ lower(and(ge(length(parameters.ginkgoSkip), 1), eq(parameters.os, 'windows'))) }}
```

Comment:
In which scenarios(from the above) we need to have this check ?

Reply:
DNS and any future tests that have do not require a ginkgo.skip to bring desired test coverage.